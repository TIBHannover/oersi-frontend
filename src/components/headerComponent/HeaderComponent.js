import React from "react"
import "./headerComponent.css"
import {withTranslation} from "react-i18next"

/**
 * HeaderComponent
 * @author Edmond Kacaj <edmondikacaj@gmail.com>
 * @param {*} props properties
 */
const HeaderComponent = (props) => {
  return (
    <header>
      <div>
        <h1>
          <b>{props.t("HEADER.TITLE")}</b>
        </h1>
        <p>
          <b>{props.t("HEADER.SUBTITLE")}</b>
        </p>
      </div>
      {props.children}
    </header>
  )
}

export default withTranslation()(HeaderComponent)
