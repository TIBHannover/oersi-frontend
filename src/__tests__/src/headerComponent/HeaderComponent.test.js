import React from "react"
import ReactDOM from "react-dom"
import HeaderComponent from "../../../components/headerComponent/HeaderComponent"

describe("HeaderComponent ==> Test UI  ", () => {
  it("HeaderComponent : should render without crashing", () => {
    const div = document.createElement("div")
    ReactDOM.render(
      <HeaderComponent>
        <h3>Test</h3>
      </HeaderComponent>,
      div
    )
    ReactDOM.unmountComponentAtNode(div)
  })
})
