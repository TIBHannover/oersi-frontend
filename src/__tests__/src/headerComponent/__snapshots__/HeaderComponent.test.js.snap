// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent ==> Test UI   HeaderComponent : should render correctly 1`] = `ShallowWrapper {}`;
